.travis.yml

```diff
@@ -30,13 +31,19 @@ before_script:
   # point to fold-grammars source directory:
   - sed -i "s#rootDir = '/vol/fold-grammars/src/'#rootDir = '`pwd`/'#" Misc/Applications/lib/foldGrammars/Settings.pm
 script:
-  - if [[ "$TEST_SUBTASK" == "default" ]]; then
+  - if [[ "$TEST_SUBTASK" == "unambig" ]]; then
     pushd .;
-    cd Misc/Test-Suite/GeorgStyle/;
-    bash run.sh ../Truth;
+    cd Misc/Test-Suite/;  
```

Comment:
you might want to create a third sub-directory called "KatharinaStyle" and move your sources into it. Thus, you don't have to worry about old code from Georg or me.